runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java

```diff
@@ -97,7 +97,8 @@ Duration getRespawnTime(int region)
 	EFH_SALT(Duration.of(9, GAME_TICKS), 0, ROCKS_33255),
 	TE_SALT(Duration.of(9, GAME_TICKS), 0, ROCKS_33256),
 	BASALT(Duration.of(9, GAME_TICKS), 0, ROCKS_33257),
-	DAEYALT_ESSENCE(Duration.of(MiningRocksOverlay.DAEYALT_MAX_RESPAWN_TIME, GAME_TICKS), 0, DAEYALT_ESSENCE_39095);
+	DAEYALT_ESSENCE(Duration.of(MiningRocksOverlay.DAEYALT_MAX_RESPAWN_TIME, GAME_TICKS), 0, DAEYALT_ESSENCE_39095),
+	DAEYALT_ROCK(Duration.of(49, GAME_TICKS), 0, DAEYALT_ROCKS, DAEYALT_ROCKS_17965, DAEYALT_ROCKS_17964);
```

Comment:
`DAEYALT_ROCKS_17965` is the depleted version of the rocks. The mineable IDs range from 17962 to 17964.